lib/elastic_record/index/documents.rb

```diff
@@ -66,6 +68,7 @@ def delete_document(id, parent: nil, index_name: alias_name)
 
         if batch = current_bulk_batch
           instructions = { _index: index_name, _id: id, retry_on_conflict: 3 }
+          instructions[:_type] = '_doc' if ElasticRecord::Version.es6?
```

Comment:
Is "_doc" necessary in ES 6.8? I know that some endpoints removed the need to use _doc between ES 6.7 and ES 6.8, such as the Get Mapping API, which no longer requires _doc to work.